gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -0,0 +1,42 @@
+"""Script to create release sites HT for v4 genomes."""
+import argparse
+import logging
+
+import hail as hl
+from gnomad.resources.grch38.reference_data import dbsnp
+
+from gnomad_qc.v4.resources.annotations import get_vep
+from gnomad_qc.v4.resources.constants import CURRENT_VERSION
```

Comment:
I know this will eventually be a complete script but lets not add the imports until they are used. This goes for argparse  and logging too.